ports/apocrunner/apocrunner/gmloader.json

```diff
@@ -0,0 +1,7 @@
+{
+    "save_dir" : "savedata",
```

Comment:
this folder doesn't exist. I know that gmloader won't create it and I think the game won't either. Can you double check ?

Reply:
Fixed.